types/ltijs/lib/Provider/Provider.d.ts

```diff
@@ -70,6 +74,8 @@ declare class Provider {
 
     onDeepLinking(_connectCallback: OnConnectCallback, options?: OnConnectOptions): true;
 
+    onUnregisteredPlatform(_requestCallback: OnRequestCallback): true;
```

Comment:
Same as above: I'd use a better fitting name here. In the original library, unregisteredPlatformCallback is used. What do you think?

Reply:
I changed it